### DISCUSSION TITLE
都退休了，没事看看绳网上有什么吧

### DISCUSSION THREAD

**Discussion by ZZZ0461** [share121/inter-knot#2269] [其他]
![IMG_20240806_002025](https://github.com/user-attachments/assets/0f2d8862-86de-4cff-a53f-ef1aef42f3d5)

**Comment by Genshin-Neuvillette** [comment]
有龙
<img src="https://bbs-static.miyoushe.com/static/2023/10/18/8979a790bf03b213a15cd6e86a068896_7816461984668282754.png" alt="那维莱特-有罪" width="75" >

**Comment by QVQ-QWQ** [comment]
派大星

**Comment by Jshuya24** [comment]
你这条龙能实现愿望吗？
![Uploading D3ABA39A6C15164B4B61CE63DA470AA8.jpg…]()

**Comment by Yith123** [comment]
楼主头像是钟离吗

**Reply by Hongxuli2006** [reply]
是啊
